ggshield/cmd/iac/scan/diff.py

```diff
@@ -22,11 +22,11 @@
     reference_option,
     staged_option,
 )
+from ggshield.cmd.utils.files import check_directory_not_ignored, is_filepath_excluded
```

Comment:
nitpick: importing `is_filepath_excluded()` like this works because `files.py` imports it, but it's an implementation detail. Would be better to keep the previous import.